typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -943,6 +943,67 @@ export const polygonzkevm: ChainMetadata = {
   },
 };
 
+export const neutrontestnet: ChainMetadata = {
+  protocol: ProtocolType.CosmWasm,
+  domainId: 33333,
+  chainId: 33333,
+  name: Chains.neutrontestnet,
+  displayName: 'Neutron Testnet',
+  displayNameShort: 'Neutron Testnet',
```

Comment:
This field is optional, no need to include it if it will be the same as displayName